pom.xml

```diff
@@ -12,7 +12,7 @@
             <dependency>
                 <groupId>org.infinispan</groupId>
                 <artifactId>infinispan-bom</artifactId>
-                <version>8.0.1.Final</version>
+                <version>8.1.0.Final</version>
```

Comment:
I believe we should update the mojo doc so that after a Final we always update the tutorials.